src/components/Question.js

```diff
@@ -3,64 +3,140 @@ import {
   IconButton,
   Typography,
   Button,
-  Box
+  InputLabel,
+  Box,
+  Select,
+  MenuItem,
+  FormControl,
+  TextField
 } from "@mui/material";
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 
-const Question = ({ question, onDelete, onEdit }) => {
+const Question = ({ question, display, onDelete, onEdit, onAnswer }) => {
+  if (display) {
+    const renderQuestionDetails = () => {
+      if (question.type === 'MC') {
+        return <>
+          <Box>Choices:</Box>
+          {question.choices.map(c => <Box>{c}</Box>)}
+        </>
 
-  const renderQuestionDetails = () => {
-    if (question.type === 'MC') {
-      return <>
-        <Box>Choices:</Box>
-        {question.choices.map(c => <Box>{c}</Box>)}
-      </>
+      } else if (question.type === 'RANGE') {
+        return <>
+          <Box>Min: {question.min}</Box>
+          <Box>Max: {question.max}</Box>
+        </>
+      } else {
+        return []
+      }
+    }
+
+    return (
+      <Box
+        key={question.id}
+        sx={{
+          padding: '10px',
+          border: '1px solid rgba(255, 255, 255, 0.35)',
+          borderRadius: '4px',
+          margin: '12px 0',
+          display: 'flex',
+          alignItems: 'center'
+        }}>
+        <Box sx={{ marginLeft: '12px' }}>{question.id}.</Box>
+        <Typography sx={{ flexGrow: 1 }}>
+          <Box sx={{ flexGrow: 1 }}>
+            <Box>Prompt: {question.question}</Box>
+            <Box>Type: {question.type}</Box>
+            {renderQuestionDetails()}
+          </Box>
+        </Typography>
+        <IconButton
+          color="inherit"
+          onClick={() => onDelete(question.id)}
+        >
+          <DeleteIcon />
+        </IconButton>
+        <IconButton
+          disabled
+          color="inherit"
+          onClick={() => onEdit(question.id)}
+        >
+          <EditIcon />
+        </IconButton>
+      </Box>
+    )
+  } else {
+    //answer question
+
+    const renderQuestionDetails = () => {
+      if (question.type === 'MC') {
 
-    } else if (question.type === 'RANGE') {
-      return <>
-        <Box>Min: {question.min}</Box>
-        <Box>Max: {question.max}</Box>
-      </>
-    } else {
-      return []
+        return <FormControl fullWidth>
+          <InputLabel id="select-label">Select an Answer</InputLabel>
+          <Select
+            label="Select an Answer"
+            onChange={e => onAnswer(e.target.value)}
+          >
+            {question.choices.map(q =>
+              <MenuItem value={q}>{q}</MenuItem>
+            )}
+          </Select>
+        </FormControl>
+
+      } else if (question.type === 'RANGE') {
+          const onRangeChange = e => {
```

Comment:
This doesn't work. Lots of buggy behavior.

Reply:
What is the specific issue? It correctly handles incorrect and correct responses in terms of range

Reply:
There was some weird behavior when trying to remove digits, latest change fixes that